src/main/java/frc/robot/Robot.java

```diff
@@ -56,7 +85,7 @@ public void disabledPeriodic() {}
   /** This autonomous runs the autonomous command selected by your {@link RobotContainer} class. */
   @Override
   public void autonomousInit() {
-    m_autonomousCommand = m_robotContainer.getAutonomousCommand();
+    // m_autonomousCommand = m_robotContainer.getAutonomousCommand();
```

Comment:
Where is your autonomousCommand set now?

Reply:
I'm not entirely sure why this was commented out, it's been uncommented now.